assets/prototype/application/ui/components/input/espressoButton/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default as EspressoButton } from './EspressoButton';
-export { ClickEvent, EspressoButtonProps, EspressoButtonSize, EspressoButtonStyle } from './types';
+export * from './types';
```

Comment:
This is the fix for this error:
![image](https://user-images.githubusercontent.com/1477453/72339777-9722ec80-36cf-11ea-91f9-cefc108b6bbf.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/72427696-fe56a480-3794-11ea-9e9d-0801d02aa76d.png)